src/css/components/_live.scss

```diff
@@ -1,19 +1,11 @@
-// We are assuming there is no progress bar and using the live display
-// to fill in the middle space. Live+DVR will need to adjust this.
 .video-js .vjs-live-control {
-  @include display-flex(flex-start);
-  @include flex(auto);
-  font-size: 1em;
-  line-height: 3em;
+  cursor: pointer;
```

Comment:
Makes the live control have a pointer, and makes it take up a similar amount of space to a button. Previously it took up all of the progress control space but the progress control will now show.

Reply:
Should it have a pointer when at the live point, if clicking will do nothing?

Reply:
hmm probably not good call.

Reply:
yes